### ISSUE TITLE
Version without username/password?

### ISSUE THREAD

**Issue by danielmgk** [And3rsL/VisonicAlarm-for-Hassio#14]
Hi,

I use the official app but without username and password. It only requires panel id and master user code. Is there anyway this amazing code can be configured to allow for a connection without username/password? I saw some one else also wrote something about a similar problem.

thanks
D

**Comment by dehooge** [comment]
Hey,
what server do you connect to? 
If your server don't require username/passsword then maybe you don't have to provide it in HA. I don't know, just guessing.

**Comment by danielmgk** [comment]
Visonics server, if you use the app and ”self service” then you only need to specify panelid and master code when you use the app visonic go. From what I have tried you need to specify all variables in this home assistant integration otherwise it will not work.

**Comment by dehooge** [comment]
We must be using different versions of this app. I can not start Visonic Go without first setting up server adress and the username/password. There is no where to click for "self service".

**Comment by danielmgk** [comment]
https://play.google.com/store/apps/details?id=com.visonic.VisonicAlert&hl=sv&gl=US

Read in the bottom of the page

**Comment by dehooge** [comment]
Yeah so you use visonic.tycomonitor.com? For me that is not working without username/password.

<p>
<img src="https://user-images.githubusercontent.com/75813403/103957802-cede3c00-514b-11eb-91db-c3042ce875ea.JPG"  height="500">
</p>

To bad you don't get the same or you could just sign up for an account and the problem would be fixed. Sorry I can't help you then.

**Comment by danielmgk** [comment]
Yes. Strange. It was created without user/pass. I will try to reinstall app then. Thanks for help.

**Comment by And3rsL** [comment]
It's very strange that your app doesnt require username/password, anyway if then you can use the older release https://github.com/And3rsL/VisonicAlarm-for-Hassio/tree/1.1.5

**Comment by danielmgk** [comment]
Hi,

Sorry it was my misstake. I must have forgot when the app was updated a e-mail/passwd was needed. Never logged out so I have entered this just once. Dehooge made me try to log out and in and I realised I actually have a registered e mail. Sorry for this confusion. It actually works now! Thanks!

I do not find the IR/movement sensors though, just the ones with camera. Saw that some one else asked about the similar issue. Perhaps not yet developed support for these sensors?

Thanks for great integration
//d
